browser/ui/views/side_panel/brave_side_panel.cc

```diff
@@ -90,6 +92,21 @@ void BraveSidePanel::UpdateWidthOnEntryChanged() {
   // Do nothing.
 }
 
+bool SidePanel::ShouldRestrictMaxWidth() const {
+  // TODO(crbug.com/394339052): Only restricting width for only non-read
```

Comment:
is this code copied from upstream? What are we copying and what is being changed here?

Reply:
It looks like it's being copied verbatim from upstream. BraveSidePanel is not subclassing SidePanel, so we probably need the same implementation as upstream. cc: @cdesouza-chromium @simonhong

Reply:
this crbug reference doesn't make sense then and not sure we should be doing the same thing based on the description

Reply:
Yes, we don't need any of that code. It wouldn't even be called. Fixed in https://github.com/brave/brave-core/pull/27522/commits/f77aff4f60c319fcfb15a079a837929b92093233